exchanges/populate/CHANGELOG.md

```diff
@@ -0,0 +1,202 @@
+# @urql/exchange-graphcache
+
+## 2.1.1
```

Comment:
Let’s trim this down to “Initial Release: Move `populateExchange` to a separate package.” and publish it as `0.1.0`? Seems like the right version to start at imho